src/data/faq.json

```diff
@@ -1186,6 +1189,9 @@
                                 "anchor": "check_in_luca_warn_for_others",
                                 "active": false,
                                 "textblock": [
+                                    "<b>Updated ?????</b>",
+                                    "This feature of the Corona-Warn-App is no longer available from version 3.0 and cannot be used with the previous versions 2.9 to 2.28 after ??????, as the verification hotline (TAN hotline) will be discontinued on this date.",

```

Comment:
same comment about "from"

Reply:
Changed in https://github.com/corona-warn-app/cwa-website/pull/3312/commits/4ffe8f2c1cf64712dae8d52554bf516c25027efa, thanks!